src/data-access/UserGroupRepository.ts

```diff
@@ -0,0 +1,18 @@
+import { Sequelize } from 'sequelize';
+import { Inject, Service } from 'typedi';
+import { UserGroup, UserGoupAttributes } from '../models';
+
+@Service()
+export class UserGroupRepository {
+    private sequelize: Sequelize;
+
+    constructor(@Inject('database') sequelize: Sequelize) {
+        this.sequelize = sequelize;
+    }
+
+    public async createAll(userGroups: UserGoupAttributes[]): Promise<UserGroup[]> {
+        return await this.sequelize.transaction(async (t) => {
```

Comment:
You don't need to use `await` when you're returning the function directly.
So you can remove it in all the cases similar.

Reply:
right, I'll fix it.